provision/acc_provision/acc_provision.py

```diff
@@ -573,8 +666,63 @@ def generate_kube_yaml(config, output):
     return config
 
 
+def generate_cf_yaml(config, output):
+    template = get_jinja_template('aci-cf-containers.yaml')
+
+    if output and output != "/dev/null":
+        outname = output
+        applyname = output
+        if output == "-":
+            outname = "<stdout>"
+            applyname = "<filename>"
+            output = sys.stdout
+        else:
+            applyname = os.path.basename(output)
+
+        info("Writing deployment vars for ACI add-ons to %s" % outname)
+        template.stream(config=config).dump(output)
+        pg = ("%s/%s" %
+              (config['aci_config']['vmm_domain']['nested_inside']['name'],
+               config['cf_config']['node_network']))
+        node_subnet = config["net_config"]["node_subnet"]
+        node_subnet_cidr = "%s/%s" % cidr_split(node_subnet)[3:]
+        node_subnet_gw = cidr_split(node_subnet)[2]
+        info("Steps to deploy ACI add-ons:")
+        # TODO Merge steps 1 & 2 into a single cloud-config update
+        info("1. Manually update your cloud config to use vCenter Portgroup " +
+             "'" + pg + "' in 'cloud_properties' of subnet " +
+             node_subnet_cidr + " in the network named " +
+             "'default'. E.g." + '''
+
+networks:
+- name: default
+  type: manual
+  subnets:
+  - range: %s
+    gateway: %s
+    [...]
+    cloud_properties:
+      name: %s
+''' % (node_subnet_cidr, node_subnet_gw, pg))
+        info("2. Update cloud config using:")
+        info("  bosh update-cloud-config <your current cloud config file> " +
+             "-o <aci-containers-release>/manifest-generation/" +
+             "cloud_config_ops.yml -l %s" % applyname)
+        info("3. Deploy ACI add-ons using:")
+        info("  bosh deploy <your current arguments> -o " +
+             "<aci-containers-release>/manifest-generation/" +
+             "cf_ops.yml -l %s" % applyname)
+
+    return config
+
+
 def generate_apic_config(config, prov_apic, apic_file):
-    apic_config = ApicKubeConfig(config).get_config()
+    configurator = ApicKubeConfig(config)
+    if config["template_type"] == TEMPLATE_CF:
+        configurator.with_kubeapi_vlan = False
+        configurator.tenant_generator = configurator.cloudfoundry_tn
+        configurator.associate_aep_to_nested_inside_domain = True
```

Comment:
This option should be moved to the config and specified in the "flavor" configuration section